.travis.yml

```diff
@@ -11,8 +11,10 @@ addons:
   firefox: latest-beta
 
 before_install:
+  # see https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI
   - "export DISPLAY=:99.0"
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
+  - sleep 3
```

Comment:
a wonky hack, but given that we don't control the environment we run in, it's duct tape that might work out ok